rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json

```diff
@@ -393,7 +393,7 @@
       {
         "key": "sample_lot_number_lbl",
```

Comment:
Edit the key names and relevance files (if applicable) to reflect the new name change

Reply:
fixed